### ISSUE TITLE
Color bar scale misplaced for zone model output (at least)

### ISSUE THREAD

**Issue by gforney** [firemodels/fds#213]
```
When visualizing zone model output with smokeview, the axis labels for the 
color bar are shifted upwards with some "falling" off the top of the 
screen after loading the zone model information.  Sample files attached.  
problem is improved, but not solved with a smokeview.ini file in the same 
directory.

Smokeview 5.0.5 downloaded directly from code website.
```

Original issue reported on code.google.com by `cfastdev` on 2007-11-29 18:41:20

<hr>
- _Attachment: standard_0027.png<br>![standard_0027.png](https://storage.googleapis.com/google-code-attachments/fds-smv/issue-234/comment-0/standard_0027.png)_
- _Attachment: [standard.smv](https://storage.googleapis.com/google-code-attachments/fds-smv/issue-234/comment-0/standard.smv)_
- _Attachment: [standard.plt](https://storage.googleapis.com/google-code-attachments/fds-smv/issue-234/comment-0/standard.plt)_

**Comment by gforney** [comment]
```
will take a look at it.
```

Original issue reported on code.google.com by `gforney` on 2007-11-29 18:52:00
- **Labels added**: Priority-Medium
- **Labels removed**: Priority-Low

**Comment by gforney** [comment]
```
I had left out a fix to the zonefire colorbar that I had applied earlier to slice, 
boundary and other colorbars.  (fixed now).  

There is still a problem with zonefire colorbar labels when the smokeview.ini file

is missing.
```

Original issue reported on code.google.com by `gforney` on 2007-11-29 20:58:09

**Comment by gforney** [comment]
```
fixed in 5.0.6 just posted.  please test and verify if you confirm that it is fixed.
```

Original issue reported on code.google.com by `gforney` on 2007-11-30 19:17:46

**Comment by gforney** [comment]
```
no report to the contrary, presume verified
```

Original issue reported on code.google.com by `gforney` on 2007-12-30 15:49:48
